### ISSUE TITLE
Question about the code in DLD

### ISSUE THREAD

**Issue by csguoh** [hikopensource/DAVAR-Lab-OCR#150]
Hi authors,

Thanks for your ownsome work of Dynamic Low-Resolution Distillation for Cost-Efficient End-to-End Text Spotting.

I have a question about the implementation about beam search. It seems that the function `beam_decode` receive the[ softmax results](https://github.com/hikopensource/DAVAR-Lab-OCR/blob/49614e490a4f67e32280e79f19670e1dd2c25d5e/davarocr/davar_distill/models/distillation/spot_res_distill.py#L171) on teacher logits, which means it is a probability not the log probability. However, in [this](https://github.com/hikopensource/DAVAR-Lab-OCR/blob/49614e490a4f67e32280e79f19670e1dd2c25d5e/davarocr/davar_distill/core/beam_search.py#L100) and [this line](https://github.com/hikopensource/DAVAR-Lab-OCR/blob/49614e490a4f67e32280e79f19670e1dd2c25d5e/davarocr/davar_distill/core/beam_search.py#L107), it seems that you have incorrectly treat the probability as log probability (although the name is logp...). I think multiply operator should be used instead of the add operator.

Best.
